modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc

```diff
@@ -12,3 +12,36 @@ However, keep in mind that the first command will fail if the target database is
 DROP COMPOSITE DATABASE inventory
 ----
 
+[[composite-databases-delete-with-aliases]]
+== Delete a composite database with constituent database aliases
+
+In general, deleting a composite database that has constituent database aliases will fail due to those database aliases.
+This affects both the local and remote constituent database aliases belonging to the specified composite database.
+
+By appending `CASCADE ALIASES` to the command `DROP COMPOSITE DATABASE`, those constituent database aliases can be dropped together with the composite database.
+This affects both the local and remote constituent database aliases belonging to the specified composite database.
+
+Using `CASCADE ALIASES` requires the `DROP ALIAS` privilege (see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[ALIAS MANAGEMENT privileges]).
+
+Given composite database and database alias:
+[source, cypher]
+----
+CREATE COMPOSITE DATABASE movies
+CREATE ALIAS movies.sweden FOR DATABASE `swedish-movies`
+----
+
+Then both the constituent database alias `sweden` and the composite database `movies` will be dropped with:
+[source, cypher]
+----
+DROP COMPOSITE DATABASE movies CASCADE ALIASES
+----
+
+This behaviour would be the same for the more general one `DROP DATABASE name` as well when using it to drop composite databases.
```

Comment:
```suggestion
This behaviour would be the same for the more general command `DROP DATABASE name` when using it to drop a composite database.
```
Also not sure about the British spelling behaviour, maybe it should be the US behavior

Reply:
it probably should yes
```suggestion
This behavior would be the same for the more general command `DROP DATABASE name` when using it to drop a composite database.
```